PerfettoCds/PerfettoCds.csproj

```diff
@@ -16,7 +16,19 @@
     <ProjectReference Include="..\Utilities\Utilities.csproj" />
   </ItemGroup>
 
+  <Target Name="DownloadTraceProcessorShell">
```

Comment:
With this change, we should consider merging the perfetto branch / code into develop and doing auto-build & basic unit tests.

Reply:
I agree. I will work on that